client/src/modules/shopping-list/index.jsx

```diff
@@ -103,8 +107,8 @@ class ShoppingList extends Component {
     if (!list) {
       return null;
     }
-    const { cohortId, description, isArchived, name, products } = list;
-    const listItems = products || [];
+    const { cohortId, description, isArchived, name } = list;
+    const listItems = items || [];
     const archivedList = listItems.filter(item => item.isOrdered);
```

Comment:
maybe this variables should contain word 'items' in name not 'list'

Reply:
I agree the current name is totally confusing - is this archived list or rather orderedItems?